From 2409846fc40f67ab34765ea744a6f1ca256bd436 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Wed, 13 Mar 2019 09:18:32 +0800
Subject: [PATCH] util/memfd.c: fixes build error

rename "static int memfd_create" to "int memfd_create".

It cuases below error on ubuntu 16.04( Ubuntu 16.04.1 LTS (GNU/Linux 4.4.0-31-generic x86_64)):

util/memfd.c:41:5: error: no previous prototype for ?memfd_create? [-Werror=missing-prototypes]
int memfd_create(const char *name, unsigned int flags)
^
cc1: all warnings being treated as errors
/root/netmap/LINUX/qemu/rules.mak:66: recipe for target 'util/memfd.o' failed
make: *** [util/memfd.o] Error 1

Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 util/memfd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/util/memfd.c b/util/memfd.c
index 4571d1a..d9c1657 100644
--- a/util/memfd.c
+++ b/util/memfd.c
@@ -37,7 +37,7 @@
 #include <sys/syscall.h>
 #include <asm/unistd.h>
 
-static int memfd_create(const char *name, unsigned int flags)
+int memfd_create(const char *name, unsigned int flags)
 {
 #ifdef __NR_memfd_create
     return syscall(__NR_memfd_create, name, flags);
-- 
2.18.0

